README.md

```diff
@@ -0,0 +1,110 @@
+# AMK-MONGO
+
+## Setup
+Set the following environment variables
+- AMK_MONGO_USERNAME
+- AMK_MONGO_PASSWORD
+- AMK_MONGO_HOSTS
+- AMK_MONGO_DEFAULT_DATABASE
+
+you can achieve this by choosing one of the options below:
+1. use [dotenv](https://github.com/motdotla/dotenv) to set the variables
+2. issue the command ``` export AMK_MONGO_USERNAME=username ```
+3. put it on the ``` .bashrc ``` or ```.bash_profile``` file
+4. for AMK_MONGO_HOSTS , it should be a string like '172.16.1.90:27017,172.16.1.91:27017,172.16.1.92:27017'
```

Comment:
`'172.16.1.90:27017,172.16.1.91:27017,172.16.1.92:27017'`. add ``

Reply:
Curious why add 3 examples?

Reply:
replica set needs a minimum of 3 mongodb instances